tests/unit/store/flask.spec.js

```diff
@@ -120,10 +120,38 @@ describe("store/flask", () => {
       expect(store.state.flask.status_uuid).not.toBe(
         STATUS.MESSAGE.CALIBRATION_NEEDED
       );
+      expect(store.state.flask.status_uuid).toStrictEqual(STATUS.MESSAGE.ERROR);
       expect(store.state.playback.playback_state).toEqual(
         playback_module.ENUMS.PLAYBACK_STATES.NOT_CONNECTED_TO_INSTRUMENT
       );
     });
+    test("Given all axios requests are mocked to return 300, When stop_status_pinging is dispatched, Then the system status updates to be in the ERROR state", async () => {
+      mocked_axios.onGet(all_mantarray_commands_regexp).reply(300);
+
+      const bound_ping_system_status = ping_system_status.bind(context);
+      await bound_ping_system_status();
+
+      await store.dispatch("flask/stop_status_pinging");
+      expect(store.state.flask.status_uuid).toStrictEqual(STATUS.MESSAGE.ERROR);
+    });
+    test("Given all axios requests are mocked to return 404, When stop_status_pinging is dispatched, Then the system status updates to be in the ERROR state", async () => {
+      mocked_axios.onGet(all_mantarray_commands_regexp).reply(404);
+
+      const bound_ping_system_status = ping_system_status.bind(context);
+      await bound_ping_system_status();
+
+      await store.dispatch("flask/stop_status_pinging");
```

Comment:
I'm not sure what you are trying to do in this test.  There is no action called `stop_status_pinging`, so this line of code does nothing. (I'm still not sure why Vuex doesn't raise an error if we attempt to invoke something that doesn't exist)

Reply:
Mistake on my side, it has to verify the following `expect(store.state.flask.status_ping_interval_id).toBe(null); expect(store.state.waveform.waveform_ping_interval_id).toBe(null);` now this is corrected.